components/atoms/LanguagePill/LanguagePill.tsx

```diff
@@ -71,7 +53,7 @@ const LanguagePill = ({ topic, classNames, onClick }: LanguagePillProps) => {
       clojure: CloJureIcon,
     };
 
-    return iconMap[name] || "";
+    return iconMap[name];
```

Comment:
Is the empty string necessary here @OgDev-01? If so, I'll put it back.

Reply:
not needed anymore with this refactor